serverless-heaven/serverless-webpack
# Upgrade dependencies and tools.

## What did you implement:

Closes #157 

## How did you implement it:

Upgraded the production and development dependencies and use `nyc` instead of the old istanbul.
Now the callstacks with crashes in the unit tests show the correct lines.
Made sure that lodash is used consistently.

Moved property setting to `beforeEach` in unit tests because Sinon 3.x does not support stubbing undefined properties anymore.

## How can we verify it:

```
npm install
npm test
npm run eslint
```

Check the `coverage` folder for coverage information.

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
